src/identity/reducers/index.ts

```diff
@@ -0,0 +1,85 @@
+// Libraries
+import produce from 'immer'
+
+import {CurrentIdentity} from '../apis/auth'
+import {RemoteDataState} from 'src/types'
+
+// Actions
+import {
+  Actions,
+  SET_QUARTZ_IDENTITY,
+  SET_QUARTZ_IDENTITY_STATUS,
+  SET_CURRENT_BILLING_PROVIDER,
+  SET_CURRENT_ORG_DETAILS,
+} from 'src/identity/actions/creators'
+
+export const initialState: CurrentIdentity = {
+  user: {
+    accountCount: 0,
+    email: '',
+    firstName: '',
+    id: '',
+    lastName: '',
+    operatorRole: null,
+    orgCount: 0,
+  },
+  org: {
+    clusterHost: '',
+    id: '',
+    name: '',
+  },
+  account: {
+    id: 0,
```

Comment:
Just making sure, the other ids are strings, this is a number. Is that correct?

Reply:
Correct, `accountId` is the odd man out - it comes back as a number
![Screen Shot 2022-06-21 at 10 45 59 AM](https://user-images.githubusercontent.com/91283923/174828896-0d95ca2c-b4a3-4d2b-abea-b8a08df75526.png)
.